### PR TITLE
[FR] After AAD working, and using Java resource endpoint, these tests works now

### PR DESCRIPTION
**Pull request by mssfang**
- [X] cause: "{"error":{"code":"InvalidImageURL","message":"Failed to download the image from the submitted URL. The URL may either be invalid or the server hosting the image is experiencing some technical difficulties."}}"
recognizeGermanContentFromUrl

- [X] cause: Analyze operation failed, errorCode: [2003], message: Download failed. Please check your input URL.
recognizeCustomFormUrlMultiPageUnlabeled
recognizeCustomFormUrlMultiPageLabeled

- [X] cause: assertNotNull(formField.getValueData().getText())
recognizeCustomFormMultiPageLabeled

Fixes partial issue in https://github.com/Azure/azure-sdk-for-java/issues/21331

### CODE REVIEW

File: sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClientTest.java [hunk 1]
```diff
@@ -1118,7 +1118,6 @@ public void recognizeCustomFormUrlUnlabeledDataIncludeFieldElements(HttpClient h
      */
     @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
     @MethodSource("com.azure.ai.formrecognizer.TestUtils#getTestParameters")
```
Comment (samvaity) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClientTest.java:1120]: recognizeGermanContentFromUrl works now too

Comment (samvaity) [reply sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClientTest.java:1120]: recognizeCustomFormMultiPageLabeled can be enabled too

**Review by kristapratico — APPROVED** [review]

**Comment by maririos** [comment]
Just to triple check, it will be good to run the live tests here to make sure everything is green

**Review by maririos — APPROVED** [review]

**Review by samvaity — APPROVED** [review]

**Pull request merged**